Jakefile.js

```diff
@@ -919,48 +921,57 @@ function getDiffTool() {
 // Baseline Diff
 desc("Diffs the compiler baselines using the diff tool specified by the 'DIFF' environment variable");
 task('diff', function () {
-    var cmd = '"' +  getDiffTool()  + '" ' + refBaseline + ' ' + localBaseline;
+    var cmd = '"' + getDiffTool() + '" ' + refBaseline + ' ' + localBaseline;
     console.log(cmd);
     exec(cmd);
-}, {async: true});
+}, { async: true });
 
 desc("Diffs the RWC baselines using the diff tool specified by the 'DIFF' environment variable");
 task('diff-rwc', function () {
-    var cmd = '"' +  getDiffTool()  + '" ' + refRwcBaseline + ' ' + localRwcBaseline;
+    var cmd = '"' + getDiffTool() + '" ' + refRwcBaseline + ' ' + localRwcBaseline;
     console.log(cmd);
     exec(cmd);
-}, {async: true});
+}, { async: true });
 
 desc("Builds the test sources and automation in debug mode");
 task("tests-debug", ["setDebugMode", "tests"]);
 
 
 // Makes the test results the new baseline
 desc("Makes the most recent test results the new baseline, overwriting the old baseline");
-task("baseline-accept", function(hardOrSoft) {
-    var files = jake.readdirR(localBaseline);
+task("baseline-accept", function () {
+    acceptBaseline("");
+});
+
+function acceptBaseline(containerFolder) {
+    var sourceFolder = path.join(localBaseline, containerFolder);
+    var targetFolder = path.join(refBaseline, containerFolder);
+    console.log('Accept baselines from ' + sourceFolder + ' to ' + targetFolder);
+    var files = fs.readdirSync(sourceFolder);
     var deleteEnding = '.delete';
     for (var i in files) {
+        var filename = path.basename(files[i]);

```

Comment:
Replace with `var filename = files[i].substr(sourceFolder.length + 1)`. As it stands, any tests in subfolders (which is a thing for projects tests!) will get baselined up to the top (and then be missing).